portal/src/components/InputFile/InputFile.component.html

```diff
@@ -1,36 +1,37 @@
-
 <div class="form__file">
-  <label
-    class="form__file_label">
-    {{ title }}
-  </label>
-  <img
-    :src="imagePath"
-    class="form__file_img"
-  >
-  <input
-    type="file"
-    :ref="refinput"
-    :accept="accept"
-    @change="onFileChange"
-    class="form__file_input"
-    name="file"
-  >
-  <span class="form__file_text">
-    {{ imagePathTxt ? imagePathTxt : $t('File-upload') }}
-  </span>
-  <div class="form__file_buttons">
-    <a
-      href="#"
-      @click.prevent="removeImage"
-      class="form__file_button _delete"
-      :class="{'_disabled': !imagePath}"
-    />
-    <a
-      href="#"
-      class="form__file_button _upload"
+  <div class="form__file_column">
+    <label
+      class="form__file_label">
+      {{ title }}
+    </label>
+    <img
+      :src="imagePath"
+      class="form__file_img"
+    >
+  </div>
+
+  <div class="form__file_column">
+
+    <div class="form__file_buttons">
+      <a
+        href="#"
```

Comment:
Het stond er al, maar springt de pagina niet naar de top als die `#` achter de url komt?

Reply:
goed punt, gek genoeg gaat het hier wel goed (bij de delete button)

Reply:
Het gaat goed met delete button, omdat daar een preventDefault op zit. https://github.com/surfedushare/search-portal/blob/1bf0be7fd0a7910541c47be45a1d644cf2922367/portal/src/components/InputFile/InputFile.component.html#L18
Dit zijn event modifiers binnen Vue: https://vuejs.org/v2/guide/events.html#Event-Modifiers

Reply:
Het vraagt geloof ik wat meer dan alleen href verwijderen. Heb je geprobeerd aan de randen van "selecteer bestand" te klikken? Het werkt nu alleen in het midden van de tekst. Ook als je href stript. 

Dit stuk code is door een niet zo groot talent geschreven ben ik bang. Je kunt even kijken of het makkelijk te fixen valt met wat simpele CSS, maar zou het dan zeker cross browser testen. Wat mij betreft mag je ook het hele component herschrijven als dat makkelijker is. De keuzes die Black Catherin heeft gemaakt zitten misschien teveel in de weg voor duidelijke simpele (cross browser werkende) code. 

Ik laat die inschatting even aan jou, maar stel gerust een vraag.

Reply:
Ik bedacht me vanochtend dat we waarschijnlijk gewoon labels willen daar i.p.v. anchors. Zie: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#Using_file_inputs

Reply:
Een veel gebruikte methode is inderdaad om de file input te hiden met css en door middel van het label de file input te triggeren